app/value_objects/contacts_search.rb

```diff
@@ -1,14 +1,14 @@
 class ContactsSearch < Searchlight::Search
   search_on Contact.includes(:contact_memberships)
 
-  searches :title, :description, :name, :organisation_id, :contact_group_id
+  searches :q, :organisation_id, :contact_group_id
 
-  def name
+  def q
     (super || "").strip
   end
 
-  def search_name
-    search.where("contacts.title LIKE :name OR contacts.description LIKE :name", name: "%#{name}%")
+  def search_q
+    search.where("contacts.title LIKE :q OR contacts.description LIKE :q", q: "%#{q}%")
```

Comment:
It is used as a wildcard. The same will happen to underscores. Other characters are escaped (I tried `'`), as per http://dev.mysql.com/doc/refman/5.6/en/mysql-real-escape-string.html

Treating `%` and `_` as wildcards doesn't seem like a significant problem. The public-facing search is hopefully going to be replaced with a finder so that wouldn't be affected after that point.